docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:22
+WORKDIR /iexec-poco-subgraph
+COPY package*.json .
+COPY schema.graphql .
+COPY subgraph.template.yaml .
+COPY networks.json .
+COPY src src
+RUN npm ci
```

Comment:
Put `npm ci` right after line 3 to use Docker cache.

Reply:
Yes thanks there it is [Move npm install to lower layer](https://github.com/iExecBlockchainComputing/PoCo-subgraph/pull/21/commits/d8ef02f6b4f4ec4cbba8e64b7bba69acd3cdc46f)!